### ISSUE TITLE
MSL eps_override AD gradient reads ~13.7% below the f64 FD referee (converged, deterministic) — the FD-noise story was masking a real systematic

### ISSUE THREAD

**Issue by bk-squared** [bk-squared/rfx#483]
Follow-up that RESOLVES the residual named in #477 ("indeterminate at f32"). The f64 referee is now measurable (DFT accumulator dtype follows x64 — PR incoming) and its pre-declared verdict band fired on the **real-AD-systematic** side.

## Measurement (MSL-FD-TIGHT objective, exact test geometry/checkpointing, CPU, main @ `7166c90` + the dtype enabler)
```
f64 AD:  g_ad = -2.110347899e-01   (matches f32 AD to ~2e-5 rel — AD is precision-stable)
f64 FD:  g_fd = -2.444668699e-01   (h=1e-3; f64 noise floor ~1e-13 → clean truncation-only)
f64 rel_err = 0.1368               (pre-declared: >=0.05 → genuine AD systematic)
```
Cross-checks that make the FD side trustworthy now:
- the f64 FD sits inside the f32 **large-h** cluster (h=3e-3 → −0.2511, h=1e-2 → −0.2422) where f32 noise is small — three independent estimates agree the true derivative ≈ **−0.244 ± 0.004**;
- AD is stable across precision (f32/f64) and platform (CPU/GPU, 5 digits) at **−0.211** → the ~13.5% deficit is in the AD tape, not in evaluation noise.

## Honest correction to the #477 close
#477's comparator-noise story explained the CPU/GPU gate straddle and remains true — but it was HALF the story. The GPU "PASS" at 0.0855 was the noisy f32 FD landing **closer to AD than the true derivative** (GPU g_fd −0.2308 vs true ≈ −0.244), i.e. comparator noise partially masked a real ~13.7% gap. The committed 0.10 gate at f32/h=1e-3 cannot resolve this — as designed it verifies "AD within 10% of a ±4%-noisy reference".

## Candidate mechanisms (named, unattributed — for the investigating session)
1. **A detached/incomplete dependency in the S assembly**: if the fitted Z0/β (N-probe wave decomposition) enter S with a broken or deliberately-detached gradient path, FD sees dZ0/dα but AD does not. (The visible `stop_gradient`s in `_sparams.py` are diagnostic-only copies — checked; the suspect is the lstsq/fit differentiability itself.)
2. `jnp.linalg.lstsq`/fit gradient correctness through the wave decomposition.
3. checkpoint_segments remat interaction with the DFT carries over 26k steps.
4. DFT windowing/timing derivative terms.

Cheap discriminators for whoever picks this up: (a) rerun the f64 referee with the objective built from RAW probe DFTs (bypassing the wave-decomposition) — if AD==FD there, the extractor fit is the culprit; (b) num_periods sweep at f64 (does the deficit grow with record length?); (c) checkpointing on/off at small num_periods f64.

## Scope warning
Every claims-bearing use of `compute_msl_s_matrix(eps_override=...)` gradients (AD==FD tests at looser tolerances, optimization runs) should treat the gradient as ~±14%-class until attributed. The forward path is unaffected. Gate untouched per `feedback_root_cause_before_gate_change`.

Repro: session scratchpad `i477_e3_f64.py` (standalone, x64-scoped process); enabled by the dtype PR. Full falsifier-tree record: #477 + `i477_falsifier_declaration.md`.

**Comment by bk-squared** [comment]
## Attributed and fixed — PR #486

**Root cause** (discriminator ladder, all pre-declared f64 referees): the auto-`eps_r_sub` branch of the MSL source builder sampled the **override** (traced) through `stop_gradient` to build the launch fixture. FD re-derived the fixture at α±h; the tape saw it frozen → FD and `jax.grad` differentiated different functions.

| Discriminator | rel_err |
|---|---|
| D1 raw forward, MSL port, auto | 61.7% |
| D2b point source (no MSL port) | 0.01% |
| D3 MSL port, explicit eps_r_sub | 0.09% |
| **V1 after fix (same D1 referee)** | **0.04–0.19%** |
| **V2 after fix (np=20 converged, full extraction — this issue's headline)** | **0.000110** (was 13.7%) |

The candidate list's #1 (detached dependency in the source/S path) was correct in class but the site was the **launch fixture**, not the wave-decomposition fit — the extraction chain (lstsq/Z0/β) is confirmed cleanly differentiable by V2. The "±14%-class" scope warning is withdrawn: the MSL `eps_override` gradient now matches the repo's best lanes.

Consequence for MSL-FD-TIGHT (#477/#479): the f32 gate's remaining content is the comparator-noise envelope only (±3–5%), so the 0.10 threshold now carries genuine margin on both platforms.

Fix + tests + docstring supersession in PR #486 (review in progress).
